serve/serve.go

```diff
@@ -99,6 +126,8 @@ func newCmdServe(opts Options) *cobra.Command {
 	cmd.Flags().StringVar(&network, "network", "tcp", `the network must be "tcp", "tcp4", "tcp6", "unix" or "unixpacket"`)
 	cmd.Flags().Var(logLevel, "log-level", fmt.Sprintf("log level. one of: %s", strings.Join(logLevel.Allowed, ",")))
 	cmd.Flags().Var(logFormat, "log-format", fmt.Sprintf("log format. one of: %s", strings.Join(logFormat.Allowed, ",")))
+	cmd.Flags().BoolVar(&noSentry, "no-sentry", false, "disable sentry")
```

Comment:
Maybe use the same wording as the CLI e.g. `disable-telemetry`?

Reply:
I think for flags it's better to use `no-` and I used it everywhere else (I don't think we have `enable` or at least shouldn't ) - this should make flags short. I can change the help to `no sentry` if this is more consistent.

Reply:
Ah sure I don't mind `no-` vs `disable-` just not use an internal implementation detail in the name of the flag